README.md

```diff
@@ -1,9 +1,536 @@
-# android-things-workshop
-AKA the connected catapult!
+# Hands-on Lab Android Things
 
-This workshop is aimed to onboard on Android Things and its API.
+This codelab will introduce you to the key concepts to creating your own IoT devices using the Android Things platform.  
+By the end of the codelab, you will build your first connected catapult and control it wirelessly.
 
-## Schematics
+Do no harm! We know how tempting it could be, but don't try to hurt other members of the codelab with your catapult. It would only bring you trouble (hint: they may retaliate with a more powerful one).
 
-![Schematics for Intel Edison](catapult-button-edison.png)
-![Schematics for Raspberry Pi 3](catapult-button-raspberry.png)
\ No newline at end of file
+## Setup
+
+Clone the project, import it to [Android Studio 2.3](https://developer.android.com/studio/index.html), and build it
```

Comment:
Let's avoid be outdated too early, do not specify the Android Studio version?
`Android Studio 2.3` -> `Android Studio`